.github/workflows/development-deploy.yml

```
@@ -66,3 +72,34 @@ jobs:
           script: |
             docker-compose -f ${{ secrets.DOCKER_COMPOSE_PATH }}/docker-compose.yml pull
             docker-compose -f ${{ secrets.DOCKER_COMPOSE_PATH }}/docker-compose.yml --env-file ${{ secrets.ENV_FILE_PATH }} up -d
+
+    check:
```

actionlint (1.7.4)

unexpected key "check" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)